src/main/java/io/jenkins/docker/DockerTransientNode.java

```diff
@@ -248,6 +249,8 @@ private void terminate(ILogger logger) {
         } catch (NotFoundException e) {
             logger.println("Container '" + containerId + "' already gone " + containerDescription + ".");
             containerNowRemoved = true;
+        } catch (ConflictException e) {
+            // container removal already in progress
```

Comment:
Probably a good idea to log this using `logger.println` so it'll show up in the logs as a low-severity issue, just like the `NotFoundException` does.